test/JDBC/expected/securityadmin_role-vu-verify.out

```diff
@@ -0,0 +1,1557 @@
+-- tsql
+alter login securityadmin_login1 with password='123'
+go
+
+alter login no_securityadmin_login1 with password='123'
+go
+
+-- make login member of securityadmin
+Alter server role securityadmin add member securityadmin_login1
```

Comment:
If a login already has a user, can it be added as a member of `securityadmin`? 
Can we add a test for the same?

Reply:
In sysadmin, we additionally added check to not allow user creation or block making member if has user mapped but here we haven't even added these additional check then why is it needed in the file? Although I have already tested it locally.